swiftsimio/initial_conditions/generate_particles.py

```diff
@@ -15,6 +15,14 @@
 from .IC_kernel import get_kernel_data
 from swiftsimio.optional_packages import KDTree, TREE_AVAILABLE
 from swiftsimio import Writer, load
+from swiftsimio.accelerated import jit
+
+@jit(nopython=True, fastmath=True)
+def correct_for_periodic(dx, ndim, boxsize):
```

Comment:
Some documentation would be great here that this is for distances, not particle positions